.github/workflows/client.yml

```diff
@@ -1,6 +1,8 @@
-name: Go
+name: Go / keep-core
 
 on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
As we have E2E tests in `local-setup` running at 0:00, I would suggest changing the cron here for `0 2 * * *`, to decrease the load on the GH Actions.

Reply:
Right, but doesn't GitHub Actions manage this sort of issues itself through some balancers? Or adding some delay to the existing job? We will need to change around 10 repos in total and potentially more in the future. Keeping track of what is scheduled when might be a bit problematic when each repo is scheduled at different times. I would leave it at 0:00 and let GH sort things out, but if you know that it would be a problem, then sure, let's change it.

Reply:
Yeah, maybe I'm just being over-cautious here - I don't know of any problems with parallel runs. You're right about keeping track of the times of cron executions, let't keep it at 0:00, we can always change it in the future if any problems occur.